docs/conf.py

```diff
@@ -50,7 +50,7 @@
     datasetfile.write("-" * (len(title) - 1) + "\n\n")
     datasetfile.write(":name: {}\n\n".format(script.name))
     datasetfile.write(":reference:  `{}`\n\n".format(reference_link))
-    datasetfile.write(":citation: {}\n\n".format(to_str(script.citation)))
+    #datasetfile.write(":citation: {}\n\n".format(to_str(script.citation)))
```

Comment:
Why are we removing this.
You could use this
datasetfile.write(":citation: {}\n\n".format((script.citation))

Reply:
@pranita-s this is comment, if you can't see the comment, I will make it a review